packages/framework/src/Model/Transport/Transport.php

```diff
@@ -12,6 +12,8 @@
 use Shopsys\FrameworkBundle\Model\Localization\AbstractTranslatableEntity;
```

Comment:
Please apply same changes as for Payment

Reply:
Yes, cahnged Payment too.